R/check-devtools.r

```diff
@@ -98,6 +98,13 @@ check_news_md <- function(pkg) {
     "NEWS.md is not ignored",
     "NEWS.md now supported by CRAN and doesn't need to be ignored."
   )
+
+  news_rd_path <- file.path(pkg$path, "inst/NEWS.Rd")
+  check_status(
+    !file.exists(news_rd_path),
+    "NEWS.Rd removed",
```

Comment:
"NEWS.Rd does not exist" would be a better message, I think.

Also this is going to be a false positive for people who use `inst/NEWS.Rd` only